pandas/core/indexes/datetimes.py

```diff
@@ -116,7 +116,7 @@ def _new_DatetimeIndex(cls, d):
     + [
         method
         for method in DatetimeArray._datetimelike_methods
-        if method not in ("tz_localize", "tz_convert")
+        if method not in ("tz_localize", "tz_convert", "strftime")
```

Comment:
`strftime` was added as an explicit method instead of automatically delegated one in https://github.com/pandas-dev/pandas/pull/36742 (the method below where I edited the return value), but at that time we forgot to correctly remove it from this list